src/commands/synthetics/run-tests-lib.ts

```diff
@@ -192,52 +168,117 @@ const getTestListBySearchQuery = async (
   }))
 }
 
-export const getTestsList = async (
+export const getFinalVersionOfTestsFromCi = async (
   api: APIHelper,
   config: RunTestsCommandConfig,
   reporter: MainReporter,
-  suites: Suite[] = []
+  suites?: Suite[]
```

Comment:
Why change this signature?

Reply:
Suits is only used in `getTestConfig` so i moved the `Suite[] = []` to there 
https://github.com/DataDog/datadog-ci/pull/1214/files#diff-d3548a697d009bf2222ed857f083b39afd505e05f34c1b3686686d7d05c6ff81R248

and made the signature of this function match the one it is called from